app/Http/routes.php

```diff
@@ -53,7 +53,7 @@
         Route::get('configuration', 'ConfigurationController@getConfiguration');
         Route::post('email', 'EmailController@sendEmail');
         Route::post('upload', 'FileUploadController@uploadFile');
-        Route::get('profiles/{id}/task', 'ProfileController@getPerformancePerTask');
+        Route::get('tasks/{ID}/performance', 'ProfileController@getPerformancePerTask');
```

Comment:
where did you see uppercase route param? :D 

Please make the param called `taskId`